tests/test_server_basic.py

```diff
@@ -44,6 +44,20 @@
                  )
 
 
+def test_meta_data_request(dispatcher_live_fixture):
+    server = dispatcher_live_fixture
+    logger.info(f"constructed server: {server}")
+
+    c=requests.get(server + "/api/meta-data",
+                   params={**default_params},
+                )
+
+    logger.info(f"content: {c.text}")
+    jdata=c.json()
+
+    assert c.status_code == 200
+
```

Comment:
can we add a test to check if log was created? because the problem was that it was not, that's why it was commented, I think

Reply:
It was crashing mainly because the job_id was not set Non for that specific case, I will add the check

Reply:
and also that it does not create a new logger for each request. That's what was happening at some points too.
I find it is convenient to inspect loggers with logging_tree, but simply logging would do too.

Reply:
it will be necessary but not sufficient, since it will not catch an problem which happened in the wild. but it might be unclear what I am trying to explain with the logging, so let me maybe to that part myself.